### ISSUE TITLE
Wrong visualization of rotated textboxes in icon/diagram

### ISSUE THREAD

**Issue by abuntrock** [OpenModelica/OpenModelica#10650]
### Description

A textbox seems to be not rotated correctly in the diagram and icon (example below with icon, same behaviour in diagram annotation). In contrast, other graphic items like rectangles are rotated correctly.

### Steps to Reproduce

Consider the following modelica class.

```modelica
model TextboxRotation
  annotation(Icon(graphics={
     Rectangle(extent={{-100,80},{100,74}},
          lineColor={225,225,225},
          fillColor={225,225,225},
          fillPattern=FillPattern.Solid),
      Text(extent={{-100,80},{100,74}},
          textColor={223,0,36},
          textString="Some horizontal text"),
      Rectangle(extent={{-75,3},{75,-3}},
          origin={0,5},
          rotation=348,
          lineColor={225,225,225},
          fillColor={225,225,225},
          fillPattern=FillPattern.Solid),
      Text(extent={{-75,3},{75,-3}},
          origin={0,5},
          rotation=348,
          textColor={223,0,36},
          textString="Some rotated text")}));
end TextboxRotation;
```

Both textboxes should be located in the same position as the two rectangles (red text should always be in front of gray rectangle) since they are defined in the same way. However, this is only the case for the horizontal textbox. The rotated textbox is not rotated correctly and its size appears to be bigger.

### Expected Behavior

Textboxes should rotate in a counter-clockwise manner as specified in the Modelica specification 18.6.1 and as already done for e.g. rectangles.

### Screenshots

My colleague @niklwors will add a screenshot of the icon below.

### Version and OS

 - OpenModelica Version: v1.22.0-dev-51-ge506fbc068 (64-bit)
 - OS: Windows 10, 64 bit

Thanks for looking into this
Aaron Buntrock - Bosch Rexroth

**Comment by niklwors** [comment]
Picture:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/12269776/236246759-3501c9f7-56b5-4639-bb89-8fdbf3cd3e94.png)

**Comment by adeas31** [comment]
Fixed with #11691 and ported to maintenance/v1.22

**Comment by abuntrock** [comment]
This now looks perfect with the current nightly. Thanks a lot
